src/features/profile/components/MyProfileGeneral.tsx

```diff
@@ -32,36 +35,47 @@ const ProfileGeneral = ({
     Object.assign(data, {
       categories: [],
       reputation: 0,
-      buyerBans: 0,
       highestFeeRecieved: { value: 0 },
       totalSalesAmount: { value: 0 },
       totalItemsSold: 0,
-      topCategoryPercentage: '0%',
     })
   }
 
   const [categories, setCategories] = useState<Array<string>>(data.categories)
   const [bio, setBio] = useState<string>(data.bio)
   const [location, setLocation] = useState<TOption | null>(locations[1])
   const [language, setLanguage] = useState<TOption | null>(languages[0])
+  const [currentPSLPrice, setCurrentPSLPrice] = useState('0')
 
   useEffect(() => {
     setLocation(locations[isEmpty ? 0 : 1])
     setLanguage(languages[0])
     setBio(isEmpty ? 'None' : data.bio)
   }, [isEmpty])
 
+  useEffect(() => {
+    const getNativeCurrency = async (): Promise<void> => {
+      if (!nativeCurrency) {
+        return
+      }
+
+      const result = await Convert(15).from('USD').to(nativeCurrency)
```

Comment:
15 is for current PSL price in USD as I can see, please move it to constant for now